src/Author.java

```diff
@@ -0,0 +1,30 @@
+public class Author {
+    String nameAuthor;
```

Comment:
1) Необходимо соблюдать инкапсуляцию. В базовом случае поля должны быть приватными
2) Свойство "имя автора" у "Автора" это масло маслянное
```suggestion
    private String name;
```